lib/widgets/subscription_list.dart

```diff
@@ -0,0 +1,239 @@
+      if (subscription.pinToTop) {
+        pinned.add(subscription);
+      } else {
+        unpinned.add(subscription);
+      }
+    }
+    // TODO(i18n): add locale-aware sorting
+    pinned.sortBy((subscription) => subscription.name);
+    unpinned.sortBy((subscription) => subscription.name);
+
+    return Scaffold(
+      appBar: AppBar(title: const Text("Streams")),
+      body: Builder(
+        builder: (BuildContext context) => Center(
+          child: CustomScrollView(
+            slivers: [
+              if (pinned.isEmpty && unpinned.isEmpty)
+                const _NoSubscriptionsItem(),
+              if (pinned.isNotEmpty) ...[
+                _SubscriptionListHeader(context: context, label: "Pinned"),
+                _SubscriptionList(unreadsModel: unreadsModel, subscriptions: pinned),
+              ],
+              if (unpinned.isNotEmpty) ...[
+                _SubscriptionListHeader(context: context, label: "Unpinned"),
+                _SubscriptionList(unreadsModel: unreadsModel, subscriptions: unpinned),
+              ],
+
+              // TODO(#188): add button to "All Streams" page with ability to subscribe
+
+              // This ensures last item in scrollable can settle in an unobstructed area.
+              const SliverSafeArea(sliver: SliverToBoxAdapter(child: SizedBox.shrink())),
+            ]))));
+  }
+}
+
+class _NoSubscriptionsItem extends StatelessWidget {
+  const _NoSubscriptionsItem();
+
+  @override
+  Widget build(BuildContext context) {
+    return SliverToBoxAdapter(
+      child: Padding(
+        padding: const EdgeInsets.all(10),
+        child: Text("No streams found",
+          textAlign: TextAlign.center,
+          style: TextStyle(
+          color: const HSLColor.fromAHSL(1.0, 240, 0.1, 0.5).toColor(),
+          fontFamily: 'Source Sans 3',
+          fontSize: 18,
+          height: (20 / 18),
+        ).merge(weightVariableTextStyle(context)))));
+  }
+}
+
+class _SubscriptionListHeader extends StatelessWidget {
+  const _SubscriptionListHeader({
+    required this.context,
+    required this.label,
+  });
+
+  final BuildContext context;
+  final String label;
+
+  @override
+  Widget build(BuildContext context) {
+    return SliverToBoxAdapter(
+      child: ColoredBox(
+        color: Colors.white,
+        child: SizedBox(
+          height: 30,
+          child: Row(
+            children: [
+              const SizedBox(width: 16),
+              Expanded(child: Divider(
+                color: const HSLColor.fromAHSL(0.2, 240, 0.1, 0.5).toColor())),
+              const SizedBox(width: 8),
+              Text(label,
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  color: const HSLColor.fromAHSL(1.0, 240, 0.1, 0.5).toColor(),
+                  fontFamily: 'Source Sans 3',
+                  fontSize: 14,
+                  letterSpacing: 0.56,
+                  height: (16 / 14),
```

Comment:
The letter spacing appears in the Figma as "4%" in some places, and "0.56px" in others. The former sounds a lot more likely to me to be the way that it's expressed in Vlad's head as the author of the design — i.e. that's the source-code form, the form one would think of it in if one was adjusting it to a different value.

So let's express it that way here: `0.04 * 14`.

This is similar to how we express the line height as `16 / 14` instead of like `1.1429`, because "16px" is the way one would think of the design when thinking of editing it.